lib/argument-store.js

```diff
@@ -67,6 +67,10 @@ class ArgumentStore {
       }
     });
 
+    if (argumentStore.get("SEMAPHORE_AGENT_STACK_NAME").length > 128) {
+      throw "SEMAPHORE_AGENT_STACK_NAME can be up to 128 characters long."
```

Comment:
It can be a good practice to show the current length as well in the error. It helps a lot in cases when other automation tools are built on top of this tool.